test-adapter/src/main/scala/org/scalajs/sbttestadapter/ScalaJSRunner.scala

```diff
@@ -181,10 +203,6 @@ final class ScalaJSRunner private[testadapter] (
     val slave = framework.createRunner(slaveLauncher)
     slave.start()
 
-    // Create a runner on the slave
-    slave.send("newRunner")
-    ComUtils.receiveLoop(slave)(msgHandler(slave) orElse ComUtils.doneHandler)
```

Comment:
Actually this is a good point. I thought this violates the contract with the test framework. While it does not do this, it might lead to unnecessary communication. But it is probably better to do it right away.